src/Standards/AcquiaPHPStrict/ruleset.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-Ruleset -->
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="../../../vendor/squizlabs/php_codesniffer/phpcs.xsd"
+         name="AcquiaPHPStrict"
+>
+
+  <description>Acquia's strict PHP coding standards.</description>
+
+  <rule ref="AcquiaPHP"/>
+
+  <!-- Drupal sniffs -->
+  <rule ref="Drupal.Commenting.InlineComment"/>
+
+  <!-- SlevomatCodingStandard sniffs -->
+  <rule ref="SlevomatCodingStandard.Arrays.AlphabeticallySortedByKeys" />
+  <rule ref="SlevomatCodingStandard.Arrays.DisallowImplicitArrayCreation"/>
+  <rule ref="SlevomatCodingStandard.Arrays.MultiLineArrayEndBracketPlacement"/>
+  <rule ref="SlevomatCodingStandard.Arrays.SingleLineArrayWhitespace"/>
+  <rule ref="SlevomatCodingStandard.Arrays.TrailingArrayComma"/>
+  <rule ref="SlevomatCodingStandard.Classes.RequireConstructorPropertyPromotion" />
+  <rule ref="SlevomatCodingStandard.Commenting.DeprecatedAnnotationDeclaration" />
+  <rule ref="SlevomatCodingStandard.Commenting.DisallowCommentAfterCode" />
+  <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing" />
+  <rule ref="SlevomatCodingStandard.Commenting.EmptyComment" />
+  <rule ref="SlevomatCodingStandard.Commenting.ForbiddenAnnotations">
+    <properties>
+      <property name="forbiddenAnnotations" type="array" value="@author,@created,@version,@package,@copyright,@license" />
```

Comment:
Thanks for allowing `@throws`, @danepowell. Did I fail to persuade you on `@package`? 🙂

Reply:
Hah, sorry... refresh my memory, what's the argument for allowing `@package`?

Reply:
The essential argument is that it allows you to designate architectural components. Some language, like Java, have the concept of a package built into the language. The `@package` annotation compensates, in a way, for the lack of such a feature in PHP.

There are static analysis tools that can help you enforce your architecture design using it. ([Deptrac](https://github.com/qossmic/deptrac) has that ability, if I remember correctly.)

But for most projects, the major benefit probably has to do with generated documentation. Tools like [phpDocumentor](https://www.phpdoc.org/) let you organize API docs by package. See an example on [their demo site](https://docs.phpdoc.org/) at the bottom of the left nav.

I use the annotation on [Composer Stager](https://github.com/php-tuf/composer-stager) (and enforce it with a custom PHPStan rule), which automatically makes it a best practice. 😉

Reply:
I found the reason we forbid these annotations: it's [what Slevomat recommends](https://github.com/slevomat/coding-standard/blob/c72b104f89c6d7bd8b739449f9f592cda2e5072b/doc/commenting.md?plain=1#L20).

It sounds like you disagree with the Slevomat assessment that one should use namespaces instead of packages? 😄

Reply:
I remove `package` from the forbidden list since it doesn't matter to me one way or the other. But if you change your mind let me know.

Reply:
I'm second guessing myself now. I wonder if something has changed since I began using it in my project--including my own thinking. Ya know what? It's easier to loosen the standard later than tighten it. Let's defer to Slevomat on this one and forbid it. I know I'm saying this after I already approved the PR, but I overlooked it before. I'll just revert the commit; you still have my approval.